doc/release-notes/0.33/0.33.md

```diff
@@ -160,6 +160,14 @@ if running on a Skylake or later processor.</td>
 <td valign="top">Upgrade to the 0.33.1 release.</td>
 </tr>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse/omr/pull/6432">#6432</a></td>
+<td valign="top">If the <tt>/proc</tt> file system is mounted with the option <tt>hidepid=2</tt> on Linux&reg; systems, the VM cannot access the <tt>/proc/1/cgroup</tt> file without the root privileges.</td>
+<td valign="top">All versions (Linux)</td>
+<td valign="top">VM fails to start when the container detection code cannot access the <tt>/proc/1/cgroup</tt> file.</td>
+<td valign="top">Remount the <tt>/proc</tt> file system with the <tt>hidepid=0</tt> option or remount the <tt>/proc</tt> file system with the <tt>gid</tt> option to allow users in a certain group to access the <tt>/proc</tt> file system.</td>
```

Comment:
```suggestion
<td valign="top"><a href="https://github.com/eclipse/omr/pull/6432">#6432</a></td>
<td valign="top">If the <tt>/proc</tt> file system is mounted with the option <tt>hidepid=2</tt> on Linux&reg; systems, the VM cannot access the <tt>/proc/1/cgroup</tt> file without root privileges.</td>
<td valign="top">All Linux versions</td>
<td valign="top">The VM fails to start when the container detection code cannot access the <tt>/proc/1/cgroup</tt> file.</td>
<td valign="top">Remount the <tt>/proc</tt> file system with the <tt>hidepid=0</tt> option or remount the <tt>/proc</tt> file system with the <tt>gid</tt> option to allow users in a certain group to access the <tt>/proc</tt> file system.</td>
```

Reply:
"All versions (Linux)" - In this "All versions"  refer to the OpenJDK version and the text in the brackets refer to the operating system.